user-guide/aragon-court/aragon-court-glossary.html

```diff
@@ -0,0 +1,124 @@
+<p>
+	<strong></strong><em>Below is a list of unique terms that you may encounter when using Aragon Court. You can learn more about these concepts and why they matter to you as an Aragon Court user by reading through the rest of the <a href="https://help.aragon.org/category/47-aragoncourt" target="_blank">Aragon Court juror documentation</a>.</em><strong><br>
+	</strong>
+</p>
+<p>
+	<strong>Appeal</strong>
+</p>
+<p>
+	When someone disagrees with the preliminary ruling of a dispute, they can challenge the outcome and escalate the dispute to the next round of jury voting. Each appeal round brings in a larger and larger group of jurors until the fourth and final appeal round, when all active jurors are drafted to vote on the final ruling.
+</p>
+<p>
+	<strong>Aragon Court</strong>
+</p>
+<p>
+	A blockchain-based dispute resolution protocol that uses randomly selected jurors to resolve subjective disputes.
+</p>
+<p>
+	<strong>Aragon Network Juror (ANJ)<br>
+	</strong>
+</p>
+<p>
+	The native token of Aragon Court, used to incentivize coherent rulings and discourage attacks against the protocol.
+</p>
+<p>
+	<strong>Arguments<br>
+	</strong>
+</p>
+<p>
+	Text, audio, video, and images put forth to make a case in favor of one of the possible dispute outcomes (Allow, Block, or Refuse to vote).
+</p>
+<p>
+	<strong>Dispute<br>
+	</strong>
+</p>
+<p>
+	When an Aragon Court subscriber believes that an Agreement they are party to has been violated and they decide to try to block the disputable action or proposal using Aragon Court.
```

Comment:
I think we should generalize this definition away from being Agreements-specific.

Technically anyone/thing can create a dispute, if they programmed the interfaces correctly.

Reply:
yeah I was trying to think about doing that but then I wondered what the arguments would be compared against and thought that pretty much every dispute that I could imagine would have some form of "agreement" that is being broken. out of curiosity, how would you word it?

Reply:
So "Agreement" I would replace with "subjective agreement", but actually I made this comment from a mis-reading of "block the disputable action or proposal".

I thought that bit was more coupled to the "disputable" architecture we've designed for Agreements, but it's actually quite close to the language we currently show in the Court Dashboard ("allow" or "block" action) so it might be 👌 .